### PR TITLE
Modify security kibana plugin name in test cluster and runner script

### PR DESCRIPTION
**Pull request by gaiksaya**
*Issue #, if available:*

*Description of changes:*
Change the security kibana plugin name to accommodate security disabled settings.
Used in setting up kibana service for IT and test clusters.

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/399850683

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/scripts/setup_runners_service.sh [hunk 1]
```diff
@@ -275,7 +275,7 @@ then
 
   if [ "$SETUP_DISTRO" = "zip" ]
   then
-    $KIBANA_ROOT/bin/kibana-plugin remove opendistro_security 
+    $KIBANA_ROOT/bin/kibana-plugin remove opendistroSecurityKibana 
     sed -i /^opendistro_security/d $KIBANA_ROOT/config/kibana.yml
```
Comment (sreekarjami) [inline .github/scripts/setup_runners_service.sh:279]: This has to be changed to sed -i /^opendistroSecurityKibana/d $KIBANA_ROOT/config/kibana.yml , as opendistro_security will no longer exist in the file.

File: .github/scripts/setup_runners_service.sh [hunk 2]
```diff
@@ -294,7 +294,7 @@ then
     docker run -d --name $DOCKER_NAME_KIBANA_NoSec --network="host" odfe-kibana-http:no-security
     docker ps
   else
-    sudo /usr/share/kibana/bin/kibana-plugin remove opendistro_security --allow-root
+    sudo /usr/share/kibana/bin/kibana-plugin remove opendistroSecurityKibana --allow-root
     sudo sed -i /^opendistro_security/d /etc/kibana/kibana.yml
```
Comment (sreekarjami) [inline .github/scripts/setup_runners_service.sh:298]: This has to be changed to sed -i /^opendistroSecurityKibana/d $KIBANA_ROOT/config/kibana.yml , as opendistro_security will no longer exist in the file.

File: .github/scripts/userdata.sh [hunk 3]
```diff
@@ -154,7 +154,7 @@ then
 sed -i "s/^echo \"cluster.name.*/echo \"cluster.name \: odfe-$ODFE_VER-$1-noauth\" \>\> \/etc\/elasticsearch\/elasticsearch.yml/g" $REPO_ROOT/userdata_$1.sh
 sed -i "/echo \"network.host/a echo \"opendistro_security.disabled: true\" \>\> \/etc\/elasticsearch\/elasticsearch.yml" $REPO_ROOT/userdata_$1.sh
 cat <<- EOF >> userdata_$1.sh
-sudo rm -rf /usr/share/kibana/plugins/opendistro_security
+sudo rm -rf /usr/share/kibana/plugins/opendistroSecurityKibana
 sudo sed -i /^opendistro_security/d /etc/kibana/kibana.yml
```
Comment (sreekarjami) [inline .github/scripts/userdata.sh:158]: This has to be changed to sed -i /^opendistroSecurityKibana/d /etc/kibana/kibana.yml , as opendistro_security will no longer exist in the file.

File: .github/scripts/userdata.sh [hunk 4]
```diff
@@ -164,7 +164,7 @@ cat <<- EOF >> userdata_$1.sh
 sudo rm -rf plugins/opendistro_security
 sed -i /^opendistro_security/d config/elasticsearch.yml
 cd /opendistroforelasticsearch-kibana/
-sudo rm -rf plugins/opendistro_security
+sudo rm -rf plugins/opendistroSecurityKibana
 sed -i /^opendistro_security/d config/kibana.yml
```
Comment (sreekarjami) [inline .github/scripts/userdata.sh:168]: This has to be changed to sed -i /^opendistroSecurityKibana/d /etc/kibana/kibana.yml , as opendistro_security will no longer exist in the file.

File: .github/workflows/staging-build-deb.yml [hunk 5]
```diff
@@ -5,425 +5,428 @@ on:
 #    - cron: '0 10 * * *'
   repository_dispatch:
     types: [staging-build-deb]
+  push:
+    branches:
+      - "test-cluster-fix"
```
Comment (sreekarjami) [inline .github/workflows/staging-build-deb.yml:11]: Please remove these lines.

File: .github/workflows/staging-build-deb.yml [hunk 6]
```diff
@@ -5,425 +5,428 @@ on:
...
-          DOCKER_USER: ${{ secrets.DOCKER_USER }}
-          DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
-        run: |
-          ES_VER=`./bin/version-info --es`
-          ODFE_VER=`./bin/version-info --od`
-          cd elasticsearch/linux_distributions
-          cd ../../..
-          cd opendistro-infra/scripts/dockerfiles/tests/elasticsearch
-          docker build --build-arg VER=$ES_VER -t opendistroforelasticsearch/elasticsearch-test-ubuntu:$ODFE_VER -f opendistro.elasticsearch.test.ubuntu.Dockerfile .
+  # plugin-availability:
```
Comment (sreekarjami) [inline .github/workflows/staging-build-deb.yml:13]: Uncomment.

File: .github/scripts/setup_runners_service.sh (hunk 1 above)
Comment (gaiksaya) [reply .github/scripts/setup_runners_service.sh:279]: The settings have not changed. Just the plugin name

File: .github/scripts/setup_runners_service.sh (hunk 2 above)
Comment (gaiksaya) [reply .github/scripts/setup_runners_service.sh:298]: The settings have not changed. Just the plugin name

File: .github/scripts/userdata.sh (hunk 3 above)
Comment (gaiksaya) [reply .github/scripts/userdata.sh:158]: The settings have not changed. Just the plugin name

File: .github/scripts/userdata.sh (hunk 4 above)
Comment (gaiksaya) [reply .github/scripts/userdata.sh:168]: The settings have not changed. Just the plugin name

**Review by sreekarjami — APPROVED** [review]

**Review by weicongs-amazon — APPROVED** [review]

**Pull request merged**